### PR TITLE
Port for libsrtp

### PR DESCRIPTION
**Pull request by sipsorcery**

### CODE REVIEW

**Comment by vicroms** [comment]
Build fails for UWP, are these failures expected?

File: ports/libsrtp/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,31 @@
+include(vcpkg_common_functions)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO cisco/libsrtp
+    REF 56a065555aea2abddaf9fb60353fe59f277837a3
+    SHA512 59afa25df79f875d28eefe95ef89b5956b1d2f319bba38ec34b832c2faa16b5425aae2f6ad19cf478afe02b28f4032b5dcf20a301d647d897d4577f66ca77376
+)
+
+set(VCPKG_C_FLAGS "${VCPKG_C_FLAGS} -D_CRT_SECURE_NO_WARNINGS /wd4703")
```
Comment (vicroms) [inline ports/libsrtp/portfile.cmake]: These lines should be inside an `if` block:

```
if (VCPKG_TARGET_IS_WINDOWS)
endif()
```

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/libsrtp/portfile.cmake (hunk 1 above)
Comment (sipsorcery) [reply ports/libsrtp/portfile.cmake]: Done.

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**